src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -384,12 +386,18 @@ IEnumerator IEnumerable.GetEnumerator()
     /// </summary>
     public sealed class SemanticVersion : IComparable, IComparable<SemanticVersion>, IEquatable<SemanticVersion>
     {
+        private const string VersionSansRegEx = @"^(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?$";
+        private const string LabelRegEx = @"^((?<preLabel>[0-9A-Za-z][0-9A-Za-z\-\.]*))?(\+(?<buildLabel>[0-9A-Za-z][0-9A-Za-z\-\.]*))?$";
+        private const string LabelUnitRegEx = @"^[0-9A-Za-z][0-9A-Za-z\-\.]*$";
+        private const string PreLabelPropertyName = "PSSemanticVersionPreLabel";
```

Comment:
Same here. We probably should use `PSSemanticVersionPrereleaseLabel`, since it's for a public property.

Reply:
Fixed.